libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3568,16 +3568,18 @@ MAV_RESULT GCS_MAVLINK::handle_command_preflight_can(const mavlink_command_long_
                     can_exists = true;
                     result = ap_kdecan->run_enumeration(start_stop) && result;
                 }
-                break;
 #else
                 UNUSED_RESULT(start_stop); // prevent unused variable error
 #endif
+                break;
             }
             case AP_BoardConfig_CAN::Protocol_Type_PiccoloCAN:
                 // TODO - Run PiccoloCAN pre-flight checks here
                 break;
             case AP_BoardConfig_CAN::Protocol_Type_UAVCAN:
+                break;
```

Comment:
@bugobliterator I'm not sure why this pair of breaks was required; falling through cases which are entirely empty shouldn't require a break or a `// FALLTHROUGH` annotation AFAIK.  Was this failing on some compiler?

Reply:
There must have been failure for one, but I might have added to all the rest, just to avoid failure just in case